develop/dev-guide-sample-application-python-mysqlclient.md

```diff
@@ -1,292 +1,284 @@
-    return cursor.fetchone()
+1. Navigate to the [Clusters](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page. 
 
+2. Click **Connect** in the upper-right corner. A connection dialog is displayed.
 
-def get_players_with_limit(cursor: Cursor, limit: int) -> List[tuple]:
-    cursor.execute("SELECT id, coins, goods FROM player LIMIT %s", (limit,))
-    return cursor.fetchall()
+3. Ensure the configurations in the connection dialog match your operating environment.
 
+    - **Endpoint Type** is set to `Public`
+    - **Connect With** is set to `General`
+    - **Operating System** matches your environment.
 
-def random_player(amount: int) -> List[tuple]:
-    players = []
-    for _ in range(amount):
-        players.append((uuid.uuid4(), 10000, 10000))
+   > **Tip:**
+   >
+   > If your program is running in Windows Subsystem for Linux (WSL), switch to the corresponding Linux distribution.
 
-    return players
+4. Click **Create password** to create a random password.
 
+   > **Tip:**
+   > 
+   > If you have created a password before, you can either use the original password or click **Reset password** to generate a new one.
 
-def bulk_create_player(cursor: Cursor, players: List[tuple]) -> None:
-    cursor.executemany("INSERT INTO player (id, coins, goods) VALUES (%s, %s, %s)", players)
+5. Run the following command to copy `.env.example` and rename it to `.env`:
 
+    ```shell
+    cp .env.example .env
+    ```
 
-def get_count(cursor: Cursor) -> None:
-    cursor.execute("SELECT count(*) FROM player")
-    return cursor.fetchone()[0]
+6. Copy and paste the corresponding connection string into the `.env` file. The example result is as follows:
 
+    ```dotenv
+    TIDB_HOST='{gateway-region}.aws.tidbcloud.com'
+    TIDB_PORT='4000'
+    TIDB_USER='{prefix}.root'
+    TIDB_PASSWORD='{password}'
+    TIDB_DB_NAME='test'
+    CA_PATH=''
+    ```
 
-def trade_check(cursor: Cursor, sell_id: str, buy_id: str, amount: int, price: int) -> bool:
-    get_player_with_lock_sql = "SELECT coins, goods FROM player WHERE id = %s FOR UPDATE"
+    Be sure to replace the placeholders `{}` with the connection parameters obtained from the connection dialog.
 
-    # sell player goods check
-    cursor.execute(get_player_with_lock_sql, (sell_id,))
-    _, sell_goods = cursor.fetchone()
-    if sell_goods < amount:
-        print(f'sell player {sell_id} goods not enough')
-        return False
+    TiDB Serverless requires a secure connection. Since the `ssl_mode` of mysqlclient defaults to `PREFERRED`, you don't need to manually specify `CA_PATH`. Just leave it empty. But if you have a special reason to specify `CA_PATH` manually, you can refer to the [TLS Connections to TiDB Serverless](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-clusters) to get the certificate paths for different operating systems.
```

Comment:
```suggestion
    TiDB Serverless requires a secure connection. Since the `ssl_mode` of mysqlclient defaults to `PREFERRED`, you don't need to manually specify `CA_PATH`. Just leave it empty. But if you have a special reason to specify `CA_PATH` manually, you can refer to the [TLS connections to TiDB Serverless](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-clusters) to get the certificate paths for different operating systems.
```

https://developers.google.com/style/capitalization#capitalization-in-references-to-titles-and-headings

> In references to any title or heading from a document that follows this guide, use sentence case even if the title or heading itself uses title case. That way, when the title or heading is eventually updated to sentence case, the reference will match.